server/opendp_apps/cypress_utils/management/commands/load_demo_data.py

```diff
@@ -0,0 +1,300 @@
+"""
+Allow deletion of data in between cypress tests
+"""
+import os
+from os.path import abspath, dirname, isdir, isfile, join
+
+from django.apps import apps
+from django.conf import settings
+from django.core.files import File
+from django.core.management.base import BaseCommand
+
+from opendp_apps.analysis.models import AnalysisPlan, DepositorSetupInfo
+from opendp_apps.cypress_utils import static_vals as cystatic
+from opendp_apps.dataset.models import UploadFileInfo
+from opendp_apps.user.models import OpenDPUser
+from opendp_apps.utils.randname import get_rand_alphanumeric
+
+DEMO_FILE_DIR = join(dirname(abspath(__file__)), 'demo_files')  # /cypress_utils/management/commands/demo_files
+
+depositor_password = os.environ.get('USER_DEPOSITOR_PASSWORD', 'Test-for-2022')
+analyst_password = os.environ.get('USER_ANALYST_PASSWORD', 'Test-for-2022')
+
+class Command(BaseCommand):
+    help = "Prepares data for demo / user testing"
+    depositor_username = 'dp_depositor'
+    analyst_username = 'dp_analyst'
+
+    def handle(self, *args, **options):
+        """Delete data in-between user tests"""
+        self.stdout.write(self.style.WARNING('>> Preparing to load demo data'))
+
+        # Important check!!
+        if not settings.ALLOW_DEMO_LOADING:  # Do not remove this check
+            self.stdout.write(self.style.ERROR(cystatic.MESSAGE_CLEAR_DATA_CMD_ERR))
+            return
+
+        if self.is_demo_data_already_loaded():
+            return
+
+        # Create users: depositor and analyst
+        #
+        depositor_user = OpenDPUser.objects.create(username=self.depositor_username,
+                                                   first_name='DP',
+                                                   last_name='Depositor',
+                                                   email='test_depositor@opendp.org')
+        depositor_user.set_password(depositor_password)
+        depositor_user.save()
+        self.write_success_msg(f'Depositor created: {depositor_user}')
+
+        analyst_user = OpenDPUser.objects.create(username=self.analyst_username,
+                                                 first_name='DP',
+                                                 last_name='Analyst',
+                                                 email='test_analyst@opendp.org')
+        analyst_user.set_password(analyst_password)
+        analyst_user.save()
+        self.write_success_msg(f'Analyst created: {analyst_user}')
+
+        # "Verify" users via email
+        #
+        VerifyModel = apps.get_model(app_label='account', model_name='emailaddress')
```

Comment:
What is the advantage of getting this via `get_model` rather than calling `'allauth.account.models.EmailAddress` directly? Seems like it makes it harder to understand exactly what is going on and what model is being used, also unclear it is coming from allauth

Reply:
Agreed. (Was in a rush and grabbed account/emailaddress from the admin url)